start.go

```diff
@@ -59,6 +62,11 @@ func readConfigFile(config_path string, flagProcfile *string, flagPort *int, fla
 	} else {
 		*flagPort = defaultPort
 	}
+	if config["shutdown_grace_time"] != "" {
+		*flagShutdownGraceTime, err = strconv.Atoi(config["shutdown_grace_time"])
```

Comment:
Errors could be silently ignored here.
Although it was already the case before, and it is not directly related to this PR,
maybe it would be better to return the error as soon as it occur instead of
having a chance to silently ignore it.